hnn_core/network_builder.py

```diff
@@ -635,6 +645,13 @@ def _record_spikes(self):
             if _PC.gid_exists(gid):
                 _PC.spike_record(gid, self._spiketimes, self._spikegids)
 
+    # setup somatic voltage recording for this node
```

Comment:
looks very clean so far! Just a tiny comment about moving this comment into a one-line docstring below. It's not so kosher to have comments above functions (I know it's there in the repo but it's mostly legacy code)

go, go, go @ntolley !!